src/typescript/frontend/src/app/launch/page.tsx

```diff
@@ -2,10 +2,17 @@ import { REVALIDATION_TIME } from "lib/server-env";
 import ClientLaunchEmojicoinPage from "../../components/pages/launch-emojicoin/ClientLaunchEmojicoinPage";
 import { isUserGeoblocked } from "utils/geolocation";
 import { headers } from "next/headers";
+import { type Metadata } from "next";
+import { emoji } from "utils";
 
 export const revalidate = REVALIDATION_TIME;
 export const dynamic = "force-static";
 
+export const metadata: Metadata = {
+  title: "launch",
+  description: `Launch your own emojicoins using emojicoin.fun ${emoji("rocket")}`,
```

Comment:
```suggestion
  description: `Launch your own emojicoins using emojicoin.fun ${emoji("rocket")}`,
```

@alnoki Ok to use the rocket emoji here?

Reply:
How about `🎉 Party Popper`?

Feel free to resolve after applying fix